test/e2e/nstemplatetier-basic.yaml

```
@@ -75,4 +75,32 @@ spec:
       parameters:
       - name: USERNAME
         required: true
-    type: stage
\ No newline at end of file
+    type: stage
+  clusterResources:
+    revision: 000000a
+    template:
+      apiVersion: template.openshift.io/v1
+      kind: Template
+      metadata:
+        labels:
+          toolchain.dev.openshift.com/provider: codeready-toolchain
+        name: advanced-cluster-resources
```

this is basic tier, not advanced:
```suggestion
        name: basic-cluster-resources
```

---

ah yes, good catch! 🤦‍♂ 
thanks!